hashicorp/dev-portal
# [chore] Remove Waypoint integrations logic

## 🔗 Relevant links

- [Preview link](https://dev-portal-git-heat-choreremove-custom-waypoin-497190-hashicorp.vercel.app/waypoint) 🔎
- [Asana task](https://app.asana.com/0/1206176289707208/1206882402102315/f) 🎟️

## 🗒️ What

- Remove custom logic related to Waypoint integrations as that content is inaccessible to users now

## 🤷 Why

- Part of the Waypoint IA restructing [project](https://app.asana.com/0/1206225787851796/1206225832503646)

## 🧪 Testing

It's kind of hard to test this because the pages have already been removed and redirects are in place, but you could test by trying to access these pages:
- Try to visit the Waypoint Integrations [page](https://dev-portal-git-heat-choreremove-custom-waypoin-497190-hashicorp.vercel.app/waypoint/integrations). It should redirect you to `/waypoint` without any errors
- Try to visit a specific Waypoint plugin [page](https://dev-portal-git-heat-choreremove-custom-waypoin-497190-hashicorp.vercel.app/waypoint/plugins/consul). It should redirect you to `/waypoint` without any errors

Otherwise, try at looking at [Well-Architected Framework](https://dev-portal-git-heat-choreremove-custom-waypoin-497190-hashicorp.vercel.app/well-architected-framework) as it uses the `serializeContent` function that was modified in this PR.

 ## 💭 Anything else?
